src/Tests/Microsoft.Diagnostics.Monitoring.Tool.FunctionalTests/Options/LoggingOptions.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Diagnostics.Monitoring.WebApi;
+using Microsoft.Extensions.Logging;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.Diagnostics.Monitoring.TestCommon.Options
+{
+    internal sealed class LoggingOptions
+    {
+        [Display(
+            ResourceType = typeof(OptionsDisplayStrings),
+            Description = nameof(OptionsDisplayStrings.DisplayAttributeDescription_LoggingOptions_LogLevel))]
+        [JsonConverter(typeof(JsonStringEnumConverter))]
+        public IDictionary<string, LogLevel> LogLevel { get; set; }
```

Comment:
```suggestion
        public IDictionary<string, LogLevel?> LogLevel { get; set; }
```

The LogLevel can be null here so that it falls back to the level set by the `Default` 'category'.